api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -115,36 +130,38 @@ public void syncBillingProjectStatus() {
           case CREATING:
           case ADDINGTOPERIMETER:
           default:
+            log.info(
+                String.format(
+                    "SyncBillingProjectStatus: BillingProject %s status=%s",
+                    entry.getFireCloudProjectName(), fireCloudStatus.toString()));
             break;
         }
       } catch (NotFoundException e) {
         log.log(
             Level.WARNING,
-            "Get BillingProjectStatus call failed for "
-                + entry.getFireCloudProjectName()
-                + ". Project not found.",
+            "Get BillingProjectStatus call failed for " + entry.toString() + ". Project not found.",
```

Comment:
Do we want the full string here? It seems to me that just the project name would be enough? (same for below)

Reply:
It's cheap enough to include the other fields, and there's a nonzero chance they could be helpful. For things that are called by crons, I like to be generous with logging context.